webpack.dev.js

```diff
@@ -0,0 +1,17 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+
+module.exports = merge(common, {
+	mode: 'development',
+	devServer: {
```

Comment:
We don't use webpack-dev-server :)
https://webpack.js.org/configuration/dev-server/